stats/Export.md

```diff
@@ -0,0 +1,34 @@
+# Export API Overview
+The stats library aggregates measurements into Views (which are essentially a collection of
+metrics, each with a different set of labels). All output objects are immutable. The core data 
+types used are:
+* `AggregationData`: contains data from an `Aggregation`.
+* `ViewData`: encapsulates `View` data aggregation.
+
+### AggregationData
+An AggregationData describes the result of data aggregation. The library SHOULD define an
+AggregationData for each Aggregation types defined.
+
+The library SHOULD provide support for multiple types of Aggregations:
+* `CountData`: data generated for a `Count` aggregation.
+* `SumDataDouble` and `SumDataInt64`: data generated for a `Sum` aggregation based on the `Measure`
+type.
+* `MeanData`: data generated for a `Mean` aggregation.
+* `DistributionData`: data generated for a `Distribution` aggregation.
+
+### ViewData
+A ViewData is defined from the following:
+* `name`: the name of the exported `View`.
+* `columns`: the `columns` of the exported `View`.
+* `Measure`: the `Measure` of the exported `View`. This field MAY be replaced with
+`MeasureDescription` if defined.
+* `aggregation`: the `aggregation` of the exported `View`
```

Comment:
I think ViewData is supposed to contain the whole View, rather than just the `name`, `columns`, `Measure`, and `aggregation` fields, so that it is completely self-contained (similar to Measure and Measurement).  That is how it is implemented in opencensus-java.

Reply:
done.